ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -346,6 +347,21 @@ - (NSString *)getCPUType {
     return typeOfCpu;
 }
 
+
+- (NSString *)getBuildId {
+    #if TARGET_OS_TV
+        return @"not available";
+    #else
```

Comment:
it can return "not available" for tvOS also - I think that's the last thing worth mentioning, since it's not just windows, maybe a global doc mention that "not available" is a possible return, and is used for both windows and tvOS

Reply:
Yes, you're right. I should mention, It's "not available" in `tvOS` but I saw the iOS bridge code and find `userAgent()` [method return "not available"](https://github.com/react-native-community/react-native-device-info/blob/9f6399b2fd6eca64ebee45b7a35edb95b06b033f/ios/RNDeviceInfo/RNDeviceInfo.m#L230-L238) for `tvOS`. Should I update this in the documentation?

Reply:
Yes please! If anything can return 'not available' but it isn't documented, that may help future people.

Reply:
Done.